geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -611,7 +611,8 @@ PartitionedRegionRedundancyTracker getRedundancyTracker() {
     return redundancyTracker;
   }
 
-  public void computeWithPrimaryLocked(Object key, Runnable r) throws PrimaryBucketLockException {
+  public boolean computeWithPrimaryLocked(Object key, Runnable r)
```

Comment:
this should no longer return boolean (since we got rid of the tryLock option)

Reply:
Good call, just updated it!